components/calendar/src/cal/chinese.rs

```diff
@@ -277,6 +294,12 @@ impl Rules for China {
 
 /// The [`Rules`] used in [Korea](https://en.wikipedia.org/wiki/Korean_calendar).
 ///
+/// # Precise definition and limits
+///
+/// This calendar is intended to represent the traditional Korean lunar calendar as used
+/// officially in the Republic of Korea as of 2025. This takes a best-effort approach
```

Comment:
also abstract over past, current, and future government forms

```suggestion
/// officially in Korea as of 2025. This takes a best-effort approach
```